.readthedocs.yaml

```diff
@@ -16,7 +17,14 @@ build:
       - playwright install chromium
       - python ./docs/scripts/generate_gallery_images.py
 
+sphinx:
+  configuration: docs/conf.py
```

Comment:
this bit is now required, and our builds will start failing in January without it.